src/crypto/ctaes/ctaes.h

```diff
@@ -1,7 +1,7 @@
  /*********************************************************************
  * Copyright (c) 2016 Pieter Wuille                                   *
  * Distributed under the MIT software license, see the accompanying   *
- * file COPYING or http://www.opensource.org/licenses/mit-license.php.*
+ * file COPYING or https://opensource.org/licenses/mit-license.php.   *
```

Comment:
Should be submitted upstream for the ctaes subtree, if wanted

Reply:
Could point, I do not know the codebase very well yet. Issued PR to ctaes in https://github.com/bitcoin-core/ctaes/pull/13 .